Project 1 (9.12)/theme switcher/switcher.js

```diff
@@ -1,2 +1,9 @@
+function darkModeClick()
+{
+document.querySelector("body").style.backgroundColor="black";
```

Comment:
jest tu jeszcze trochę do zrobienia w tej funkcji. Jak wystawiasz pull request, to ja uznaję, że zadanie jest zrobione :)